cinema/views.py

```diff
@@ -1,13 +1,15 @@
 from datetime import datetime
 
 from django.db.models import F, Count
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import extend_schema, OpenApiParameter
 from rest_framework import viewsets, mixins, status
 from rest_framework.authentication import TokenAuthentication
```

Comment:
do you need this and other imports?
In the task you should add JWT auth + use it in all views

Reply:
i don't understand, OpenApiParameter and extend_schema i need for description, OpenApiTypes i used for DATE type